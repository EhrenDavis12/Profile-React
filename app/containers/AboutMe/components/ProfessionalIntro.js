// import PropTypes from 'prop-types';
import React from 'react';
// import HeaderBar from 'components/HeaderBar';
import { P, H2, Row12, PLeft } from './styles';

function ProfessionalIntro() {
  return (
    <div className="row">
      <Row12>
        <H2>Professional Intro</H2>
      </Row12>
      <Row12>
        <PLeft>
          Experienced Software Engineer with a demonstrated history of working
          in the computer software industry. Strong engineering professional
          skilled in Bash scripting, Automation tools, and Problem solving.
        </PLeft>
        <P>
          Software Engineer II at LawLogix Group, Inc.
          <br />
          Employed: May 2013 – PresentEmployment
          <br />
          located in Phoenix, AZ 85012
          <br />
        </P>
        <PLeft>
          Responsible for re-engineering, maintaining and error handling of an
          automated file batch upload system running daily transaction for 80+
          different clients. Reduced implementation by 60% and proactively
          decreased errors and client inquiries by 30%. Responsible for ensuring
          the proper handling of client sensitive information (PII). Review,
          Design, improve, and automate client data import process. Improved
          data imported accuracy by 42% and decreased steps by 33%. Utilized
          SQL, system logs, and created additional quality assurance
          techniques/tools and steps to ensure validations in identifying issues
          or answering client questions within same day SLA. Developed
          enhancements to the code base based on client’s SOW requirements.
        </PLeft>
      </Row12>
    </div>
  );
}

export default ProfessionalIntro;
