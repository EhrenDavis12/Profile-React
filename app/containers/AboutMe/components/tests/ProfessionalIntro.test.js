import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import ProfessionalIntro from '../ProfessionalIntro';

describe('<ProfessionalIntro />', () => {
  it('should render the page', () => {
    const renderedComponent = shallow(<ProfessionalIntro />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
