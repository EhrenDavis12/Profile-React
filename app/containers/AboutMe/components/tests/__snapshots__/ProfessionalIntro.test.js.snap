// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfessionalIntro /> should render the page 1`] = `
<div>
  <Header>
    Professional Intro
  </Header>
  <div>
    <styles__PLeft>
      Experienced Software Engineer with a demonstrated history of working in the computer software industry. Strong engineering professional skilled in Bash scripting, Automation tools, and Problem solving.
    </styles__PLeft>
  </div>
</div>
`;
